src/Microsoft.DotNet.Darc/DarcLib/Constants.cs

```diff
@@ -33,4 +35,19 @@ public class Constants
     public const string EngFolderName = "eng";
     public const string CommonScriptFilesPath = $"{EngFolderName}/common";
     public const string DefaultCommitAuthor = "dotnet-maestro[bot]";
+
+    /// <summary>
+    /// Available update frequencies for subscriptions.  Currently the enumeration values aren't available
+    /// through the generated API client.  When/if they ever are, this can be removed.
+    /// </summary>
+    public static readonly List<string> AvailableFrequencies =
```

Comment:
Can you make sure this list is re-used from other places? For instance the same list is in `Microsoft.DotNet.Darc.Constants` so we can remove it from there.
Also I can see it's in more places around the codebase. Not sure how it's accessible from project reference point of view.